evennia/contrib/grid/ingame_map_display/tests.py

```diff
@@ -0,0 +1,35 @@
+"""
+Tests of ingame_map_display.
+
+"""
+
+
+from evennia.commands.default.tests import BaseEvenniaCommandTest
+from evennia.utils.create import create_object
+from typeclasses import rooms, exits
+from . import ingame_map_display
+
+
+class TestIngameMap(BaseEvenniaCommandTest):
+    """
+    Test the ingame map display by building two rooms and checking their connections are found
+
+    Expected output:
+    [ ]--[ ]
+    """
+    def setUp(self):
+        super().setUp()
+        self.west_room = create_object(rooms.Room, key="Room 1")
+        self.east_room = create_object(rooms.Room, key="Room 2")
+        create_object(exits.Exit, key="east", aliases=["e"], location=self.west_room, destination=self.east_room)
+        create_object(exits.Exit, key="west", aliases=["w"], location=self.east_room, destination=self.west_room)
+
+    def west_room_map_room(self):
```

Comment:
These tests will not run - each test method must start with the string `test_`. Doing so and running this test suite shows that these tests actually both fail.

Reply:
Fixed!